src/client/templates/cardModal.html

```diff
@@ -44,7 +44,7 @@ <h4 class="modal-title">{{text}}</h4>
                     <form id="commentFormField">
                         <div class="commentFormGroup input-group">
                             <span class="input-group-addon text-success hidden-xs" for="Comment">Comment</span>
-                            <input class="form-control" type="text" id="comment" placeholder="Leave a comment." maxlength="400">
+                            <input class="form-control" type="text" id="comment" placeholder="Leave a comment." maxlength="500">
```

Comment:
The character limits are hard-coded, but it's important to have a character limit at ALL